platform/s3ip-sysfs/scripts/s3ip_sysfs_conf.json

```diff
@@ -0,0 +1,76 @@
+{
+  "s3ip_syfs_paths": [
+    {
+      "path": "/sys_switch/temp_sensor",
+      "type" : "path",
+      "value" : "/sys/s3ip/temp_sensor",
+      "description": "温度传感器信息"
+    },
+    {
+      "path": "/sys_switch/vol_sensor",
+      "type" : "path",
+      "value" : "/sys/s3ip/vol_sensor",
+      "description": "电压传感器信息"
+    },
+    {
+      "path": "/sys_switch/syseeprom",
+      "type" : "path",
+      "value" : "/sys/s3ip/syseeprom",
+      "description": "ONIE EEPROM"
+    },
+    {
+      "path": "/sys_switch/fan",
```

Comment:
What if a platform drivers are already loaded and path to read fan_speed is 
/sys/bus/i2c/devices/x-00y/fan<>_input

How can we mention this in the JSON file? Can you give an example?

Reply:
The example you want is shown below,
{
      "path": "/sys_switch/fan/fan1/motor0/speed",
      "type" : "path",
      "value" : "/sys/bus/i2c/devices/x-00y/fan<>_input",
      "description": "the motor0 of fan1 speed information"
    },